c/driver/sqlite/sqlite_test.cc

```diff
@@ -189,12 +189,13 @@ class SqliteStatementTest : public ::testing::Test,
   void SetUp() override { ASSERT_NO_FATAL_FAILURE(SetUpTest()); }
   void TearDown() override { ASSERT_NO_FATAL_FAILURE(TearDownTest()); }
 
-  void TestSqlIngestUInt64() { GTEST_SKIP() << "Cannot ingest UINT64 (out of range)"; }
-  void TestSqlIngestBinary() { GTEST_SKIP() << "Cannot ingest BINARY (not implemented)"; }
-  void TestSqlIngestTimestampTz() {
```

Comment:
All of the sqlite datetime functions use UTC internally, so I think reasonable to just let the implementation discard the tz https://www.sqlite.org/lang_datefunc.html